.pipelines/e2e.yaml

```diff
@@ -25,6 +25,7 @@ pool:
 
 jobs:
 - job: Run_AgentBaker_E2E
+  timeoutInMinutes: 90
```

Comment:
we already specify a timeout below, do we need this as well? we also want to make sure to keep the local timeout the same (e.g. what's in e2e-local.sh)

Reply:
if that's the case can we just get rid of the timeout from the `go test ... ` cmd? should be fine if we're allowing more than 60 minutes for these suites, and less places to have to update the better

Reply:
It seems the default timeout for go test is 10 minutes, so I think we need to keep both values. But I can make a variable so that it uses only one value

Reply:
ah didn't realize there was a default timeout, makes sense

either way is fine then